recipe/common.php

```diff
@@ -58,11 +58,20 @@
         $composer = run('which composer')->toString();
     }
 
-    if (empty($composer)) {
-        run("cd {{release_path}} && curl -sS https://getcomposer.org/installer | {{bin/php}}");
+    if (!empty($composer)) {
+        return $composer;
+    }
+
+    // check for a locally installed version of composer
+    $localFile = env()->parse('{{release_path}}/composer.phar');
+    if (is_file($localFile)) {
```

Comment:
**Deployer** will check file is existed in remote server. Please using check block instead `is_file`
```
if (run("if [ -f $localFile ]; echo 'true'; endif")->toBool()) {
    // TODO
}
```